Weapon/src/main/java/dk/sdu/se/f23/InVasion/weapon/WeaponPlugin.java

```diff
@@ -1,22 +1,39 @@
 package dk.sdu.se.f23.InVasion.weapon;
 
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import dk.sdu.se.f23.InVasion.common.data.Entity;
 import dk.sdu.se.f23.InVasion.common.data.GameData;
 import dk.sdu.se.f23.InVasion.common.data.ProcessAt;
 import dk.sdu.se.f23.InVasion.common.data.World;
 import dk.sdu.se.f23.InVasion.common.services.EntityProcessingService;
 import dk.sdu.se.f23.InVasion.common.services.PluginService;
 
+import java.util.ArrayList;
+import java.util.Objects;
+
 public class WeaponPlugin implements PluginService {
     private String weaponName = "Steven";
```

Comment:
What?
```suggestion
    private String weaponName = "";
```

Reply:
Its just a placeholder name, if you dont like it, i dont kow what to tell you ;)